lib/model.js

```diff
@@ -2802,6 +2801,8 @@ Model.findByIdAndRemove = function(id, options) {
  *
  * @param {Array|Object} docs Documents to insert, as a spread or array
  * @param {Object} [options] Options passed down to `save()`. To specify `options`, `docs` **must** be an array, not a spread. See [Model.save](https://mongoosejs.com/docs/api/model.html#Model.prototype.save()) for available options.
+ * @param {Boolean} [options.ordered] saves the docs in series rather than parallel.
+ * @param {Boolean} [options.immediateError] If a Error occurs, throw the error immediately, instead of collecting in a result, default: true (backwards compatability)
```

Comment:
I'm not sure I like this option name. I was thinking `continueOnError` to be consistent with `eachAsync()` and `createCollections()`, but `continueOnError` aggregates an error object to throw when an error occurs. Do you have any other naming ideas? Maybe `useAllSettled` or `aggregateErrors`?

Reply:
i had named it `immediateError` because of the current (master) behavior where once a error is encountered the first one is directly thrown / rejected, but all operations *should* still run and any subsequent error is just silent

Reply:
Maybe we name the option `aggregateErrors` instead? I think that options that are treated as `false` by default are cleaner to work with.